resources/sass/layout/_header.scss

```diff
@@ -0,0 +1,157 @@
+header {
+    width: 100%;
+    nav {
+        -webkit-box-shadow: 0 8px 6px -6px rgba(0, 0, 0, .15);
+        -moz-box-shadow: 0 8px 6px -6px rgba(0, 0, 0, .15);
+        box-shadow: 0 8px 6px -6px rgba(0, 0, 0, .15);
+    }
+    .row-header {
+        @include div-row;
+        text-align: center;
+        ul>li {
+            margin-left: 20px;
+            a {
+                text-decoration: none;
+                color: #787677;
+                font-weight: 600;
+            }
+        }
+        .logo {
+            flex: 0;
+            height: 70px;
+            width: auto;
+        }
+        .main-nav {
+            flex: 1;
+            display: flex;
+            align-items: center;
+            margin-left: 11%;
+            margin-top: 3%;
+        }
+        .super-header {
+            flex: 0;
+            font-size: 14px;
+            .cart-section {
+                margin-top: 21%;
+                display: flex;
+                background-color: #eeeeee;
+                width: 100px;
+                align-items: center;
+                justify-content: center;
+                .img {
+                    width: 40px;
+                    height: 40px;
+                }
+            }
+        }
+    }
+}
+
+.header-nav {
+    display: flex;
+    list-style: none;
+}
+
+.show {
+    display: none;
+}
+
+.hide {
+    display: block;
+}
+
+.hide-menu {
```

Comment:
OMG, you really don't know how to name classes, they are all doing exact opposite of what they are named for 👎